ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -726,11 +726,34 @@ FEDataManager::spread(const int f_data_idx,
     TBOX_ASSERT(cc_data || sc_data);
 
     // Make a copy of the Eulerian data.
-    const int f_copy_data_idx = var_db->registerClonedPatchDataIndex(f_var, f_data_idx);
+    int depth;
+    if (cc_data)
+    {
+        Pointer<CellDataFactory<NDIM, double> > f_cc_fac = f_cc_var->getPatchDataFactory();
+        depth = f_cc_fac->getDefaultDepth();
+    }
+    if (sc_data)
+    {
+        Pointer<SideDataFactory<NDIM, double> > f_sc_fac = f_sc_var->getPatchDataFactory();
+        depth = f_sc_fac->getDefaultDepth();
+    }
+    const f_data_idx_map_key_type f_key{ cc_data ? 0 : 1, depth };
```

Comment:
I am sure that this works, but I had to think really hard about operator precedence between the ternary expression and the comma operator. Could you put the first bit in parentheses?

Reply:
Addressing the above comment will address this one as well.